backend/check_user_roles.py

```
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
```

Ruff (0.13.3)

Shebang is present but file is not executable

(EXE001)